src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>leaderboard</title>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
+      crossorigin="anonymous"
+    />
+</head>
+<body>
+    <div class="container">
+        <h1>Leaderboard</h1>
+        <div class="row">
+          <div class="col">
+            <h3>
+              Recent scores
+              <button id="refresh" type="button" class="btn btn-light border border-dark">
+                Refresh
+              </button>

```

Comment:
- [ ] Kindly move the button to the right of the container based on the feedback from the previous reviewer. This is in keeping with the wireframe design provided.

  ![image](https://user-images.githubusercontent.com/36542986/217540962-c5ffb427-231b-4c1f-9b8a-26e0cfe5c422.png)